{"frames":[{"func":"merge(foo, bar, by = \"a\")","file":"","line_number":0,"end_line_number":0,"character_number":0,"end_character_number":0},{"func":".handleSimpleError(function (cond) \n.Internal(C_tryCatchHelper(addr, 1L, cond)), \"object 'foo' not found\", \n    base::quote(merge(foo, bar, by = \"a\")))","file":"","line_number":0,"end_line_number":0,"character_number":0,"end_character_number":0},{"func":"h(simpleError(msg, call))","file":"","line_number":0,"end_line_number":0,"character_number":0,"end_character_number":0}],"message":"Error in h(simpleError(msg, call)) : \n  Fehler bei der Auswertung des Argumentes 'x' bei der Methodenauswahl f√ºr Funktion 'merge': object 'foo' not found\n"}